WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -614,24 +615,16 @@ private extension ProductFormViewController {
     }
 
     func configureTooltipPresenter() {
-        guard ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productDescriptionAIFromStoreOnboarding) else {
-            return
-        }
-
-        guard aiEligibilityChecker.isFeatureEnabled(.description) else {
-            return
-        }
-
-        if let tooltip = tooltipPresenter?.tooltip {
-            tooltip.removeFromSuperview()
+        if let tooltipPresenter {
+            tooltipPresenter.removeTooltip()
             self.tooltipPresenter = nil
         }
 
-        guard product.description?.isEmpty == true else {
+        guard aiEligibilityChecker.isFeatureEnabled(.description) else {
```

Comment:
❓ I might have missed this earlier, does it make sense for this condition to be included in the use case's `shouldShowTooltip`?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/10073/commits/5ced25e6bd2951a0971829912cbb95b55c6b8058

I had this earlier and reverted the change to avoid initializing when `ProductDescriptionAITooltipUseCase` when AI is not eligible. I missed adding it back when I made the variable lazy. 👍